src/tsjson.test.ts

```diff
@@ -306,6 +306,30 @@ describe("More involved tests with objects", () => {
     const toFail = {};
     expect(() => parser.parse(JSON.stringify(toFail))).toThrow();
   });
+
+  test("Object disallowing additionalProperties", () => {
```

Comment:
Let me know if you want me to add another test where the schema defined `additionalProperties: false`, but the `removeAdditional` option is omitted. I thought that might be overkill since that's more towards testing ajv's internals, which we may not need to worry about here.

Reply:
We'll take this conversation to #30